boot.js

```diff
@@ -164,68 +170,25 @@ Boot.prototype.after = function (func, cb) {
   return this
 }
 
-Boot.prototype.ready = function (func) {
-  this._readyQ.push(func)
+Boot.prototype.onClose = function (func) {
+  this._closeQ.push(func, err => {
+    if (err) this._error = err
+  })
```

Comment:
I think we should call [`unshift()`](https://www.npmjs.com/package/fastq#unshift) here. More or less, the latest you register a shutdown part, the sooner it will be executed. This is the inverse of the start phase.